sdcm/utils/raft/common.py

```diff
@@ -131,26 +141,114 @@ def run_bootstrap_and_abort_with_action(self, terminate_pattern, abort_action: C
         LOGGER.debug("Clear stop event for wait_for on node %s", self.bootstrap_node.name)
         self.bootstrap_node.stop_wait_db_up_event.clear()
 
+    def _rebootstrap_node(self):
+        self.bootstrap_node.start_scylla_server(verify_up_timeout=3600, verify_down=True)
+        self.bootstrap_node.start_scylla_jmx()
```

Comment:
not needed anymore, unless backporting to branch with jmx

Reply:
it should backported to 6.0, 6.1 and 6.2 and 2024.2. I  think 6.0 still use jmx? any way start_scylla_jmx already has verification for jmx

Reply:
we don't backport to 6.0 anymore

Reply:
we can leave it in for now